tao_compiler/mlir/disc/IR/hlo_disc_ops.cc

```diff
@@ -700,6 +700,12 @@ LogicalResult SparseSegmentMeanOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// CustomCallV2Op
+//===----------------------------------------------------------------------===//
+
+LogicalResult CustomCallV2Op::verify() { return Verify(*this); }
```

Comment:
What are the differences between CustomCallV2 and CustomCallV1?

Reply:
V2 defines more attributes. I added a TODO to merge these two version once the refactor is done.